splash/jsutils.py

```diff
@@ -39,6 +40,15 @@ def escape_js(*args):
             else if (o instanceof Date) {
                 return o.toJSON();
             }
+            else if (o instanceof ClientRect) {
+                return {
+                    top: o.top, left: o.left, bottom: o.bottom, right: o.right,
+                    width: o.width, height: o.height,
```

Comment:
Someone may want to assign a value to client rect in order to bypass sanitanization.